src/eos-crash-metrics.c

```diff
@@ -41,6 +68,22 @@ report_crash (const char *binary,
   g_variant_dict_insert_value (&dict, "ostree_commit", g_variant_new_string (ostree_commit));
   g_variant_dict_insert_value (&dict, "ostree_url", g_variant_new_string (ostree_url));
 
+  if (info->app && info->runtime)
```

Comment:
`info` is not checked if it is `NULL` here, I think that is what was intended. Though perhaps it may be better to just pass the `FlatpakInstalledRef` instances directly into this function